tests/yam/agama/auto.pm

```diff
@@ -8,12 +8,25 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
-
+use testapi;
+use utils 'clear_console';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    assert_screen('agama-installing', 120);
+
+    assert_screen('agama-install-finished', 1200);
+
+    $testapi::password = 'linux';
+
+    select_console 'root-console';
+
+    clear_console;
+
+    enter_cmd "reboot";
```

Comment:
Should we just push the reboot button on following page? 
https://openqa.opensuse.org/tests/3285898#step/auto/3
Or if we are using _auto_ scenario then the reboot process should also automatic, so maybe an issue we should submit.

Reply:
> Should we just push the reboot button on following page? https://openqa.opensuse.org/tests/3285898#step/auto/3Should we just push the reboot button on following page? https://openqa.opensuse.org/tests/3285898#step/auto/3

we don't want to interact with the UI without Playwright, only for sync screens.